GeneratorAnalysis/Scripts/GeneratorInteBreakDown.cpp

```diff
@@ -46,8 +46,8 @@ void GeneratorInteBreakDown() {
     std::vector<TString> Process{"","QE","MEC","RES","DIS"};
     std::vector<int> Colors{kBlack,kBlue,kRed+1,kOrange+7,kGreen+1,kMagenta+1};
 
-    Names.push_back(OutFilePath+"FlatTreeAnalyzerOutput_GENIE_AR23.root"); 
-    Labels.push_back("GENIE_AR23");
+    /*Names.push_back(OutFilePath+"FlatTreeAnalyzerOutput_GENIE_AR23.root"); 
+    Labels.push_back("GENIE_AR23");*/
```

Comment:
One last thing, can we make it so that the original list of names/labels for the generators remains the same?